src/encoded/__init__.py

```diff
@@ -144,8 +145,13 @@ def main(global_config, **local_config):
     # set google reCAPTCHA keys
     settings['g.recaptcha.key'] = os.environ.get('reCaptchaKey')
     settings['g.recaptcha.secret'] = os.environ.get('reCaptchaSecret')
+
     # set mirrored Elasticsearch location (for staging and production servers)
-    settings['mirror.env.name'] = get_mirror_env_from_context(settings)
+    mirror = get_mirror_env_from_context(settings)
+    if mirror is not None:
+        settings['mirror.env.name'] = mirror
+        settings['mirror_health'] = get_health_page(ff_env=mirror)
```

Comment:
At some point we'll need another name for this than `ff_env` because I think this will actually work for cgap, too, but I understand why that name is in there for now and adjustments would be complex. So this is fine and looks way more robust.

Reply:
Yeah, that type of change would require a major version update for dcicutils as that keyword argument is littered throughout our codebase.